### DISCUSSION TITLE
How to config Dokodemo-door Tunnel in order to use it as a Relay server on Hiddify? | چگونه می‌توان سرور ریلی با روش Dokodemo-door ایجاد کرد و از آن در هیدیفای استفاده نمود؟

### DISCUSSION THREAD

**Discussion by lymanjre** [hiddify/Hiddify-Manager#3442] [Q&A سوالات رایج] [answered]
How to config Dokodemo-door Tunnel in order to use it as a Relay server on Hiddify? 

 چگونه می‌توان سرور ریلی با روش Dokodemo-door ایجاد کرد و از آن در هیدیفای استفاده نمود؟

**Comment by ReturnFI** [comment, marked as answer]
# جهت مشاهده مقاله آپدیت شده در خصوص این روش [اینجا](https://hiddify.com/fa/manager/domain-worker-cdn-and-tunneling/Tutorial-for-setting-up-Dokodemo-Door-tunnel-as-relay-server/) را ببینید.

**Comment by Majidygh** [comment]
با سلام . در این روش تنها پورت 443 تانل میشه، برای اضافه کردن سایر پورت ها چه باید کرد ؟

**Reply by ReturnFI** [reply]
سلام.
قسمت "تعریف چند inbound مختلف در سرور میانی برای اتصال به چند سرور اصلی" مطالعه بفرمایید توضیح داده شده با مثال پورت 8443 
موفق باشید

**Comment by HamedABC66** [comment]
برای ipv6 چه کنم ؟ جای ipv4 از ipv6 توی دستورها جاگذاری کنم ؟

**Reply by ReturnFI** [reply]
سلام.
کافیه بجای IPv4 از IPv6 استفاده کنید

**Reply by Ataheritar** [reply]
پیش نیازها برای تانل ipv6 چی هستند؟ آیا سرور ایران باید ipv6 داشته باشه؟ اگه جواب مثبت هست کجای دستورات ازش استفاده میشه؟
ممکنه یه آموزش کامل با ipv6 قرار بدین؟

**Reply by ReturnFI** [reply]
درود،
یکسری روش ها در یوتیوب هستش
که با استفاده از iproute2 و ساختن فایل در netplan برای سرور IPv6 لوکال ایجاد میکنن و از IPv6 لوکال استفاده میکنن
ضمنا روش IPv6 لوکال زمانی لازم شما میشه که سرور ایران شما IPv6 نداشته باشه.
درصورتی که سرور ایران شما IPv6 داره کافیه بجای MAIN_SERVER_IP از IPv6 سرور خارج استفاده کنید.

**Comment by GoldenOne1** [comment]
کدی که برای تک اینباند قرار دادین به درستی کار کرد ولی کد مولتی اینباند مشکل داره،لطفا یه بررسی میفرمایید ببینید کجای کدی که قرار دادید مشکل داره؟

**Reply by ReturnFI** [reply]
سلام وقتتون بخیر
از اسکریپت زیر استفاده کنید.
https://github.com/hiddify/hiddify-relay

**Reply by GoldenOne1** [reply]
سلام وقت شما هم بخیر،حقیقتا فکر نمیکردم به این زودی جواب بدید،گفتم شاید طول بکشه،این اسکریپت مشکل مولتی اینباند نداره؟ میخوام 2 تا اینباند بزنم

**Reply by ReturnFI** [reply]
نه مشکلی نداره میتونی با گزینه add another inbounds که گذاشتم اینکارو انجام بدید.
🕊️هیدیفای، راهکار آزادی اینترنت🕊️
